src/app/shared/dropdown/dropdown.component.html

```diff
@@ -0,0 +1,17 @@
+<form [formGroup]="searchGroup" class="form" autocomplete="off">
+    <input formControlName="searchControl"
+           [(ngModel)]='currentSearch'
+           (keyup)="handleOnInputChange($event)"
```

Comment:
Don't mix template and reactive forms. Instead of(keyup) use .valueChages(), instead od [(ngModel)] - get/set value right into form control using setValue()